web/src/filter.ts

```diff
@@ -702,6 +702,11 @@ export class Filter {
         });
     }
 
+    is_in_home(): boolean {
+        // All messages view.
+        return this._terms.length === 1 && this.has_operand("in", "home");
+    }
```

Comment:
`filter.ts` usually has 100% unit test coverage; would be good to test this.

Reply:
Added a test.